rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbIT.java

```diff
@@ -0,0 +1,497 @@
+
+        // Second vote for an unauthorized caller
+        voteFeePerKb(seventhFeePerKbVote, FeePerKbVoteCaller.UNAUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Third vote from authorizer 3, same value as the first one voted from authorizer 1
+        voteFeePerKb(seventhFeePerKbVote, FeePerKbVoteCaller.THIRD_AUTHORIZED.getRskAddress());
+        currentFeePerKb = seventhFeePerKbVote;
+
+        // Get fee per kb, should return the third vote value
+        assertFeePerKbValue(seventhFeePerKbVote);
+    }
+
+    @Test
+    @Order(8)
+    void voteFeePerKbChange_whenAuthorizerOneAndThreeVoteAfterAuthorizerOneVoteAuthorizerTwoDifferentValueAfterThatUnauthorizedCallerVotesSameThanAuthorizerTwoAndLasVoteIsEmittedByAuthorizerThree_shouldSetValueVotedByOneAndThree() {
+        Coin eighthFeePerKbVote = Coin.valueOf(40_000L);
+
+        // First vote for a new value
+        voteFeePerKb(eighthFeePerKbVote, FeePerKbVoteCaller.FIRST_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Second vote for a new different value
+        voteFeePerKb(differentFeePerKbVote, FeePerKbVoteCaller.SECOND_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Third vote is same than the second vote value, but it is an unauthorized caller
+        voteFeePerKb(differentFeePerKbVote, FeePerKbVoteCaller.UNAUTHORIZED.getRskAddress());
+        //It still has the previous voting round value, since it shouldn't change with an unauthorized caller vote
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Fourth vote from authorizer 3, same value as the first one voted from authorizer 1
+        voteFeePerKb(eighthFeePerKbVote, FeePerKbVoteCaller.THIRD_AUTHORIZED.getRskAddress());
+        currentFeePerKb = eighthFeePerKbVote;
+
+        // Get fee per kb, should return the third vote value
+        assertFeePerKbValue(eighthFeePerKbVote);
+    }
+
+    @Test
+    @Order(9)
+    void voteFeePerKbChange_whenUnauthorizedCallerAndAuthorizerTwoVoteSameValueThenNotChangeValueAndAuthorizerOneAndTwoVoteSameValue_shouldSetValueVotedByOneAndThree() {
+        Coin ninthFeePerKbVote = Coin.valueOf(90_000L);
+
+        // First vote is by an unauthorized caller
+        voteFeePerKb(differentFeePerKbVote, FeePerKbVoteCaller.UNAUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Second vote is by authorizer 1
+        voteFeePerKb(ninthFeePerKbVote, FeePerKbVoteCaller.FIRST_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Third vote is by authorizer 2. it votes similar then the unauthorized caller
```

Comment:
nit: `Third vote is by authorizer 2, who votes the same as the unauthorized caller`

Reply:
(i mean, the vote is not _similar_ but the same)

Reply:
Done!